src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpDoNotUseStackallocInLoops.cs

```diff
@@ -7,12 +7,13 @@
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Operations;
+using Microsoft.NetCore.Analyzers;
 
-namespace Microsoft.NetCore.Analyzers.Runtime
+namespace Microsoft.NetCore.CSharp.Analyzers.Runtime
 {
     /// <summary>CA2014: Do Not Use Stackalloc In Loops.</summary>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
-    public sealed class DoNotUseStackallocInLoopsAnalyzer : DiagnosticAnalyzer
+    public sealed class CSharpDoNotUseStackallocInLoopsAnalyzer : DiagnosticAnalyzer
```

Comment:
I don't know if there is any preference between having a language specific (syntax) analyzer being fully in the language layer or still having the base class in the common layer. I went with 1st option here.

Reply:
Let's switch to option 2 to help ensure all AnalyzerReleases are in the core layer.

Reply:
That's ok, I will do some changes to get to this direction.